lib/nrf_modem_lib/Kconfig

```diff
@@ -24,12 +24,37 @@ config NRF_MODEM_LIB_SYS_INIT
 
 config NRF_MODEM_LIB_TRACE_ENABLED
 	bool
-	prompt "Enable proprietary traces over UART"
+	prompt "Enable proprietary traces"
+	help
+	  The default size of the Trace region is 16384 bytes.
```

Comment:
I think it'd rather make sense to mention that the Trace region size is specified with `NRF_MODEM_LIB_SHMEM_TRACE_SIZE` config, the default value might be a subject of change in the future (and I doubt anyone will remember to update this help string).

Reply:
Ok done